tests/TopN.CoreFX.issues.json

```diff
@@ -0,0 +1,5 @@
+[
+    "System.Collections.Tests",
+    "System.Collections.Immutable.Tests",
+    "System.Net.Security.Tests"
```

Comment:
Could you please delete `"System.Net.Security.Tests"` from here. Instead, add `System.Runtime`, `System.Runtime.Extensions`, `System.Threading`, `System.Threading.Tasks`.

Reply:
Agree - "System.NET.Security.Tests" needs to be deleted as per previous discussions.
I would prefer to see more tests to be added as a separate PR once we know that this change works reliably.